website/static/js/filepage/revisions.js

```diff
@@ -140,7 +140,7 @@ var FileRevisionsTable = {
                     return m('.alert.alert-warning', {style:{margin: '10px'}}, model.errorMessage);
                 }
 
-                return m('table.table', [
+                return m('table.table', {style: {tableLayout: 'fixed'}}, [
```

Comment:
@sloria - thoughts on applying inline CSS this way?

My initial reaction is that this should be a class, or a more general CSS rule should be used to fix this issue. That said, inline CSS is added elsewhere in this file.